docs/open-source-tizen/tools/mic/mic-customize-image.md

```diff
@@ -1,49 +1,55 @@
+You can specify which repository must be used to create a package, including a local repository.
+
+For example: [http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min...](http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min-pc/ivi-min-pc-tizen_20120926.2.ks)
 
 ```bash
 repo --name=Tizen-main --baseurl=https://download.tizen.org/snapshots/trunk/common/@BUILD_ID@/repos/main/armv7l/packages/ --save  --ssl_verify=no
- 
+
 repo --name=Tizen-base --baseurl=https://download.tizen.org/snapshots/trunk/common/@BUILD_ID@/repos/base/armv7l/packages/ --save  --ssl_verify=no
- 
-repo --name=mylocal--baseurl=/loca/repo/path/ 
+
+repo --name=mylocal--baseurl=/local/repo/path/
 ```
 
-## How to specify user/password to access repo
+## Specifying the User and Password to Access the Repository
 
-If the remote repository access requires a password, then you can specify user/passwdto for the repo in the ks file. For example:
+If remote repository access requires a password, you can specify the user and password for the repository in the `.ks` file:
+
+```
+repo --name=REPO-NAME --baseurl=https://username:passwd@yourrepo.com/ia32/packages/ --save  --ssl_verify=no
+```
 
-repo --name=REPO-NAME --baseurl=[https://username:passwd@yourrepo.com/ia32/packages/](https://username:passwd@yourrepo.com/ia32/packages/) --save  --ssl_verify=no
+## Adding and Removing Packages
 
-## How to add / remove packages
+You can specify the packages which you plan to install in the `%packages` section of the `.ks` file. Packages can be specified by a group/pattern or by an individual package name. The definition of the group/pattern can be referred to in the `repodata/*comps.xml` or `repodata/pattern.xml` file, which are both released on the download server.
 
-You can specify the packages which you plan to install in the '%packages' section in the ks file. Packages can be specified by group/pattern or by individual package name. The definition of the groups/pattern can be referred to in the repodata/*comps.xml or repodata/pattern.xml file, which are released on the download server. For example: [http://download.tizen.org/snapshots/trunk/common/latest/repos/base/ia32/packages/repodata/](http://download.tizen.org/snapshots/trunk/common/latest/repos/base/ia32/packages/repodata/). The %packages section is required to end with '%end'. Also, multiple '%packages' sections are allowed. Additionally, individual packages may be specified by using globs. For example:
+For example: [http://download.tizen.org/snapshots/trunk/common/latest/repos/base/ia32/packages/repodata/](http://download.tizen.org/snapshots/trunk/common/latest/repos/base/ia32/packages/repodata/).
```

Comment:
Link is broken

Reply:
I know the link is broken.
mic page will be obsolete soon so I don't want to change mic.
If you want to change the link, you can use http://download.tizen.org/snapshots/tizen/base/latest/repos/standard/packages/repodata/.

Reply:
It is legacy. As I mentioned before, mic will be obsoleted. Before then we just leave them.

Reply:
OK, no changes